src/seedu/addressbook/commands/ClearCommand.java

```diff
@@ -11,6 +11,10 @@
 
     public static final String MESSAGE_SUCCESS = "Address book has been cleared!";
 
+    public boolean isMutating() {
```

Comment:
You should add override notation here to help other people know. Besides that, if its behaviour is different from others, a header comment is required as well.